connectors/jazzradio.js

```diff
@@ -9,12 +9,35 @@ new class extends BaseConnector {
         super();
         this.name = 'JazzRadio';
         this.prefix = '/com/jazzradio';
-        this.playButtonSelector = '.icon-play';
```

Comment:
I'd use `#play-button a` selector as `this.playButtonSelector` and remove `playPause` getter. `#play-button a` works like a play/pause toggler.

Reply:
But this selector is not `click()`able as far as my tests tell me. I've implemented a working `playPause` in https://github.com/f1u77y/web-media-controller/pull/20/commits/afd24a2a26f9b9a5434f0a7d1b10e4c64ed80845 .

Reply:
That's odd. The selector works for me. Your solution works fine too, though.

Reply:
@doronbehar could you say what do your tests exactly do? `#play-button a` works for me too.

Reply:
Oh pardon me. I missed the trailing `a` when I tested that initially - meaning the selector matched the `<div>` and not the `<a>`. Would you like me to work this out using _this_ selector? I won't mind but I don't see much benefit in that..

Reply:
Of cource it could be left as is, that's why I didn't unresolve the conversation. I was just interested what could go wrong.